### PR TITLE
Ticket7350

### PR DESCRIPTION
**Pull request by NikolaRoev**
https://github.com/ISISComputingGroup/IBEX/issues/7350

### CODE REVIEW

File: tjmpapSup/tjmpap.proto [hunk 1]
```diff
@@ -0,0 +1,18 @@
+getIDN {
+    out "*IDN?";
+    in "%\$1[^\r\n]";
+    ExtraInput = Ignore;
+}
+
+cmd {
+    out "\$1";
+}
+
```
Comment (FreddieAkeroyd) [inline tjmpapSup/tjmpap.proto]: Is a Db file missing? I couldn't see anywhere that used this protocol file

File: tjmpapSup/tjmpap.template [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+record(mbbi, "$(P)JMP:MODE")
+{
+    field(INP, "$(PVPREFIX)TJMPER_$(TJMPER_IOC_ID):MODE CP MSS")
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZRST, "All out")                  # All home
+    field(ONST, "PLT1 and SMPL engaged")    # Engage Plate 1 and sample
+    field(TWST, "PLT2 and SMPL engaged")    # Engage Plate 2 and sample
+    info(interest, "HIGH")
+}
```
Comment (LilithCole) [inline tjmpapSup/tjmpap.template:30]: When does this record actually process? SCAN is Passive but I don't see anything linking to it

File: tjmpapSup/tjmpap.template [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+record(ai, "$(P)SAMPLE:TEMP")
+{
+    field(INP, "$(PVPREFIX)$(SAMPLE_TEMP_READBACK):TEMP CP MSS")
+    field(DESC, "Value for the sample temperature")
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    info(interest, "HIGH")
+    info(archive, "VAL")
+}
```
Comment (LilithCole) [inline tjmpapSup/tjmpap.template:51]: Likewise for this input

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# iocBoot depends on all *App dirs
+iocBoot_DEPEND_DIRS += $(filter %App,$(DIRS))
+
+# Add any additional dependency rules here:
+
+include $(TOP)/configure/RULES_TOP
+
+ioctests:
+    .\system_tests\run_tests.bat
```
Comment (LilithCole) [inline Makefile]: makefile requires a tab indentation here

**Review by LilithCole — CHANGES_REQUESTED** [review]

File: tjmpapSup/tjmpap.template (hunk 2 above)
Comment (NikolaRoev) [reply tjmpapSup/tjmpap.template:30]: Doesn't using `CP` in the `INP` field mean that the record is processed when the input updates?

**Review by LilithCole — APPROVED** [review]

**Pull request merged**